.scss-lint.yml

```diff
@@ -0,0 +1,183 @@
+  # Prefer hexadecimal color codes over color keywords.
+  ColorKeyword:
+    enabled: true
+  # Prefer color literals to be used only in variable declarations.
+  ColorVariable:
+    enabled: true
+  # Disabled - Prefer // comments over /* ... */.
+  Comment:
+    enabled: false
+  # Reports @debug statements (which you probably left behind accidentally).
+  DebugStatement:
+    enabled: true
+  # Rule sets should be ordered as follows:
+  # @extend declarations, @include declarations without inner @content, properties,
+  # @include declarations with inner @content, then nested rule sets.
+  DeclarationOrder:
+    enabled: true
+  # Reports when you define the same property twice in a single rule set.
+  DuplicateProperty:
+    enabled: true
+  # Place @else statements on the same line as the preceding curly brace.
+  ElsePlacement:
+    enabled: true
+  # Separate rule, function, and mixin declarations with empty lines.
+  EmptyLineBetweenBlocks:
+    enabled: true
+  # Reports when you have an empty rule set.
+  EmptyRule:
+    enabled: true
+  # Files should always have a final newline.
+  FinalNewline:
+    enabled: true
+  # You can specify whether you prefer shorthand or long-form hexadecimal colors
+  # by setting the style option to short or long, respectively.
+  HexLength:
+    style: long
+  # Checks if hexadecimal colors are written in lowercase/uppercase.
+  HexNotation:
+    style: lowercase
+  # Ensure hexadecimal colors are valid
+  HexValidation:
+    enabled: true
+  # Avoid using ID selectors.
+  IdSelector:
+    enabled: true
+  # Avoid using !important in properties.
+  ImportantRule:
+    enabled: true
+  # The basenames of @imported SCSS partials should not begin with an underscore
+  # and should not include the filename extension.
+  ImportPath:
+    enabled: true
+  # Use two spaces per indentation level.
+  Indentation:
+    character: space
+    width: 2
+  # Write leading zeros for numeric values with a decimal point like 0.5em
+  LeadingZero:
+    style: include_zero
+  # Reports when you define the same selector twice in a single sheet.
+  MergeableSelector:
+    enabled: true
+  # Functions, mixins, variables, and placeholders name formats
+  NameFormat:
+    allow_leading_underscore: true # Allow names to start with a single underscore
+    convention: camel_case
+  # Avoid nesting selectors too deeply.
+  NestingDepth:
+    max_depth: 3
+  # Always use placeholder selectors in @extend
+  PlaceholderInExtend:
+    enabled: true
+  # Limit the number of properties in a rule set.
+  PropertyCount:
+    enabled: true
+    max_properties: 12
+  # Sort properties in a strict order.
+  PropertySortOrder:
+    enabled: false
+  # Reports when you use an unknown CSS property
+  PropertySpelling:
+    enabled: true
+  # Configure which units are allowed for property values.
+  PropertyUnits:
+    global: ['em', 'rem', '%', 'px'] # Allow relative units globally
+    properties:
+      font-size: ['rem']
```

Comment:
Unfortunately its stupid as hell:
`.element {
  font-size: 10px;
}
`
Reports:
`test.scss:8 [W] PropertyUnits: px units not allowed on `font-size`; must be one of (rem)
test.scss:8 [W] VariableForProperty: Property font-size should use a variable rather than a literal value`
but
`
$size: 10px;

.element {
  font-size: $size;
}
`
Does not report any error.

I will remove this linter ok ? @mr-mig